### DISCUSSION TITLE
Sign-up open!

### DISCUSSION THREAD

**Discussion by The-Compiler** [pytest-dev/sprint#6] [Announcements]
We just finished the sign-up form, thanks for everyone's patience!

Please [sign up](https://app.formbricks.com/s/clvtdzv8706zdh818vndks441) as soon as possible if you want to join.

**Comment by flub** [comment]
I hate to give you more work, any chance the landing page of the signup could say which things are going to be asked?  It's nice to know if I'll be able to answer everything, and maybe I still need to think about some of the things asked.

**Reply by The-Compiler** [reply]
Good point though. I passed it on to my contact at Omicron who created the survey. In the meantime, the list in #1 is a pretty close approximation.

**Comment by webknjaz** [comment]
@The-Compiler form feedback: it doesn't have an e-mail input, nor does it have an estimate on getting any communication back. I assume the communication method will be tagging the GitHub handle somewhere in discussions, right?

**Reply by The-Compiler** [reply]
Yup, unfortunately the fact that we should ask for an e-mail didn't make it from my paper notes to #1 somehow... Since it's only a handful of people, I'll probably try to find people's mails via their GitHub profiles, as I agree that would be easiest for replies.

Good point regarding ETA as well. I'll try to get things finalized in the upcoming week. Given that we only have a handful of sign-ups and most of them are from central Europe, I'd expect that we can invite everyone who wants to join at this point.

**Reply by webknjaz** [reply]
@The-Compiler nice, thanks for the update! I was mentioning the sprint during PyCon US, but in general the comms seem very narrow and I learned of it by accident from @hugovk's Mastodon (having registered on that platform just a week before).
I feel like some folks from the US would be open to participating. When I mentioned it to @jaraco, he was happy to join-in remotely but was sad that it's only in-person. Also, some people from places where travel tends to be expensive, plan their international trips in advance. So I feel like even if they were to request funding but received approvals late, they'd not be able to book anything cheap enough. Some people that I know are great engineers seemed interested but weren't sure how they'd contribute so they didn't even try filling it out..
Do you think, the event would benefit from more applications? Perhaps, asking some big accounts like EuroPython would help, WDYT?

**Reply by The-Compiler** [reply]
> @The-Compiler nice, thanks for the update! I was mentioning the sprint during PyCon US, but in general the comms seem very narrow and I learned of it by accident from @hugovk's Mastodon (having registered on that platform just a week before).

Eh, I've:

- Added it to pytest.org back in January
- Sent mails to the pytest and testing-in-python mailinglists back then
- Mentioned this at PyConDE, both in my pytest training and at the event announcements
- Mentioned this on GitHub and Twitter/X in various places (unfortunately not via `@pytestdotorg` as I lost access due to X requiring mail verification, and I'm still waiting for the person who originally created the account to change the mail...)

I feel like I'm out of options, but "very narrow" isn't how I'd describe things after exhausting pretty much all possibilities I have...

> I feel like some folks from the US would be open to participating. When I mentioned it to @jaraco, he was happy to join-in remotely but was sad that it's only in-person. 

I do want to open things up for remote participation as well. I agree that hasn't really been communicated, but that's mostly because I want to focus on in-person participants first.

> Also, some people from places where travel tends to be expensive, plan their international trips in advance. So I feel like even if they were to request funding but received approvals late, they'd not be able to book anything cheap enough. Some people that I know are great engineers seemed interested but weren't sure how they'd contribute so they didn't even try filling it out..

Yeah, we're kind of too late for international travel at this point, and that's partially also because the sign-up opened quite late. Unfortunately things have been unexpectedly busy ever since March for me, and my hope to get some additional pytest core maintainers to help with organizing stuff didn't really work out very well.

> Do you think, the event would benefit from more applications? Perhaps, asking some big accounts like EuroPython would help, WDYT?

I think it'd be better to keep this year's sprint at it's 5-10 people size, also because it's the first time and we're still figuring a lot of things out (I've never really organized an event of this scale, and neither has my contact at Omicron afaik). I'm hoping to repeat it next year, and use everything we've learned to make it bigger and better. Having a sign-up that's open much earlier to allow for international travel will definitely be on that list!

**Reply by webknjaz** [reply]
So my perception was wrong, then. Thanks for the info!

**Comment by The-Compiler** [comment]
**Sign-up is now closed.**

Please [reach out](https://github.com/the-compiler) if you missed the sign-up, we might have spots available on short notice. There is also the possibility of [remote participation](https://github.com/pytest-dev/sprint/discussions/8), but the number of slots will be limited.
